underfs/b2/src/main/java/alluxio/underfs/b2/B2UnderFileSystem.java

```diff
@@ -0,0 +1,286 @@
+   * @throws ServiceException
+   */
+  public static B2UnderFileSystem createInstance(AlluxioURI alluxioURI,
+      UnderFileSystemConfiguration conf) throws ServiceException {
+    LOG.debug("initializing B2 Storage Client");
+    LOG.debug(alluxioURI.toString());
+    B2StorageClient client = B2StorageClientFactory.createDefaultFactory()
+        .create(conf.get(PropertyKey.B2_ACCESS_KEY), conf.get(PropertyKey.B2_SECRET_KEY),
+            USER_AGENT);
+
+    try {
+      B2AccountAuthorization accountAuthorization = client.getAccountAuthorization();
+      LOG.debug(accountAuthorization.toString());
+
+      B2Allowed allowed = accountAuthorization.getAllowed();
+      LOG.debug(allowed.toString());
+
+      String accountId = accountAuthorization.getAccountId();
+      List<String> capabilities = allowed.getCapabilities();
+
+      String bucketNameFromURI = UnderFileSystemUtils.getBucketName(alluxioURI);
+      LOG.debug("bucketname from uri: {}", bucketNameFromURI);
+
+      String bucketId;
+      String bucketName;
+      if (allowed.getBucketId() != null) {
+        bucketId = allowed.getBucketId();
+        bucketName = allowed.getBucketName();
+      } else {
+        B2Bucket bucket = client.getBucketOrNullByName(bucketNameFromURI);
+        LOG.debug(bucket.toString());
+        bucketId = bucket.getBucketId();
+        bucketName = bucket.getBucketName();
+      }
+
+      short bucketMode = B2Utils.translateBucketAcl(capabilities);
+
+      return new B2UnderFileSystem(alluxioURI, conf, client, bucketId, bucketName, bucketMode,
+          accountId);
+    } catch (B2Exception | NullPointerException e) {
+      LOG.error("Failed to instantiate B2UnderFileSystem client with key:{} and URI: {}",
+          conf.get(PropertyKey.B2_ACCESS_KEY), alluxioURI);
+      Throwables.propagateIfPossible(e, ServiceException.class);
+    }
+    return null;
+  }
+
+  protected B2UnderFileSystem(AlluxioURI alluxioURI, UnderFileSystemConfiguration conf,
+      B2StorageClient b2StorageClient, String bucketId, String bucketName, short bucketMode,
+      String accountId) {
+    super(alluxioURI, conf);
+    requireNonNull(alluxioURI, "alluxioURI is null");
+    requireNonNull(conf, "conf is null");
+    requireNonNull(b2StorageClient, "b2StorageClient is null");
+    requireNonNull(bucketId, "bucketId is null");
+    requireNonNull(bucketName, "bucketName is null");
+    requireNonNull(bucketMode, "bucketMode is null");
+    requireNonNull(accountId, "accountId is null");
+
+    LOG.debug("alluxioURI: {}", alluxioURI);
+    LOG.debug("buckedId: {}", bucketId);
+    LOG.debug("bucketName: {}", bucketName);
+    LOG.debug("bucketMode: {}", bucketMode);
+    LOG.debug("accountId: {}", accountId);
+
+    mB2StorageClient = b2StorageClient;
+    mBucketId = bucketId;
+    mBucketName = bucketName;
+    mBucketMode = bucketMode;
+    mAccountId = accountId;
+  }
+
+  @Override public boolean createEmptyObject(String key) {
+    return false;
```

Comment:
All of our codebase follows the pattern where the annotation `@Override` is on a line above the method definition. Could you update all of these methods so that they conform?

Reply:
Fixed